### ISSUE TITLE
The BlackbirdSql extension is still in preview. When will it be officially released?

### ISSUE THREAD

**Issue by BlackbirdSQL** [BlackbirdSQL/Firebird-DDEX-SqlEditor#16]
I would like to install the BlackbirdSql extension, but it is still in preview. When will it be officially released?

**Comment by BlackbirdSQL** [comment]
All features that were earmarked for the first official release have been implemented.

Barring the discovery of any major new bugs, release of the extension is set for **Friday, 9 February 2024**.

**Comment by ErikEJ** [comment]
Wow, looking forward to this - will it be free, and can I include in EF Core Power Pack?

**Comment by BlackbirdSQL** [comment]
> Wow, looking forward to this - will it be free, and can I include in EF Core Power Pack?

Yes and yes. Just be aware the extension has only been tested on the stock Visual Studio IDE. 
For this release I cannot say if it will function or even install on Code.

**Comment by ErikEJ** [comment]
Thanks! EF Core Power Pack is not a Code extension pack, it is a VS extension pack
